gherkin/ruby/lib/gherkin/id_generator.rb

```diff
@@ -0,0 +1,35 @@
+require 'securerandom'
+
+module Gherkin
+  class IdGenerator
+    @@use_incrementing ||= false
```

Comment:
Please avoid using class variables. They add global state to the system, which we want to avoid.